libs/dapp-config/src/chains/evm/index.tsx

```diff
@@ -175,4 +143,83 @@ export const chainsConfig: Record<number, ChainConfig> = {
     evmRpcUrls: ['https://alpha-rpc.scroll.io/l2'],
     logo: EtherLogo,
   },
+
+  // Self hosted chains
+  [PresetTypedChainId.HermesOrbit]: {
+    chainType: ChainType.EVM,
+    group: 'eth',
+    chainId: EVMChainId.HermesOrbit,
+    name: 'Hermes Orbit',
+    base: 'webb-dev',
+    tag: 'test',
+    url: 'https://hermes-testnet.webb.tools',
+    evmRpcUrls: ['https://hermes-testnet.webb.tools'],
+    blockExplorerStub: 'https://hermes-explorer.webb.tools',
+    logo: GanacheLogo,
+    env: ['development', 'test'],
+  },
+  [PresetTypedChainId.AthenaOrbit]: {
+    chainType: ChainType.EVM,
+    group: 'eth',
+    chainId: EVMChainId.AthenaOrbit,
+    name: 'Athena Orbit',
+    base: 'webb-dev',
+    tag: 'test',
+    url: 'https://athena-testnet.webb.tools',
+    evmRpcUrls: ['https://athena-testnet.webb.tools'],
+    blockExplorerStub: 'https://athena-explorer.webb.tools',
+    logo: GanacheLogo,
```

Comment:
I guess we need to use the new logo here?

Reply:
The logo here is deprecated, we are using the dynamic `ChainIcon` component to render the chain logo based on the chain name.